{
  "id": "CVE-2024-53507",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2024-11-29T20:15:21.027",
  "lastModified": "2024-12-02T17:15:13.707",
  "vulnStatus": "Awaiting Analysis",
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "A SQL injection vulnerability was discovered in Siyuan 3.1.11 in /getHistoryItems."
    },
    {
      "lang": "es",
      "value": "Se descubri\u00f3 una vulnerabilidad de inyecci\u00f3n SQL en Siyuan 3.1.11 en /getHistoryItems."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
          "baseScore": 9.8,
          "baseSeverity": "CRITICAL",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "HIGH",
          "availabilityImpact": "HIGH"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 5.9
      }
    ]
  },
  "weaknesses": [
    {
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
      "type": "Secondary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-89"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://github.com/siyuan-note/siyuan/issues/13057",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/siyuan-note/siyuan/issues/13077",
      "source": "cve@mitre.org"
    }
  ]
}